framework/src/application/node/node.ts

```diff
@@ -669,6 +669,31 @@ export class Node {
 				);
 			},
 		);
+
+		// eslint-disable-next-line @typescript-eslint/no-misused-promises
+		this._chain.events.on(
+			EVENT_VALIDATORS_CHANGED,
+			// eslint-disable-next-line @typescript-eslint/no-misused-promises
+			async (
```

Comment:
this `async` and the eslint-disable is not needed

Reply:
Thought we were making all these async as a pattern in the past. Will remove.

Reply:
Changed to sync. removed all eslint comments